Update rename.R

moutikabdessabour opened the pull request:
Solved issue for column names containing spaces issue #109

moutikabdessabour commented on R/pull.R:
```diff
@@ -22,15 +22,15 @@ pull. <- function(.df, var = NULL) {
 #' @export
 pull..data.frame <- function(.df, var = NULL) {
 
-  .df <- as_tidytable(.df)
```
removed to enhance performance

moutikabdessabour commented:
added number argument feature and some speed enhancements to `pull.`

markfairbanks commented:
@moutikabdessabour - Thanks for taking a look at these!

Can you do a couple things:
1) Could you do a new pull request for the changes you made to `rename.()` only?
2) Can you add a test to that pull request to show it fixes the issue? Current `rename.()` tests can be found [here](https://github.com/markfairbanks/tidytable/blob/master/tests/testthat/test-rename.R)

I'm still weighing options on the `pull.()` changes you sent over, but I'd like to merge `rename.()` in the meantime

moutikabdessabour commented:
I had removed changes from the `pull.` and added a test for the new feature. BTW using a `data.frame` coerces spaced column names to be spaced by periods ie `spaced name` => `spaced.name` if the you have not set the `check.names` argument to `False`. Other than that the function works as expected both with `data.frame(... , check.names=F)` and with a `tidytable` and a `data.table`.

NB : the test is only for a data.table but it's the same behaviour for all types as there is only `rename..data.frame`

markfairbanks commented:
This looks great - thanks for the help!

markfairbanks commented:
I added you as a contributor to `tidytable` before the last release, but completely forgot to confirm I had your name right.

Is the name "Abdessabour Moutik" correct?

It shows up [here](https://cran.r-project.org/web/packages/tidytable/tidytable.pdf) in the package documentation.

moutikabdessabour commented:
Thanks you got it right.
